src/CircleGraph/CircleGraph.ts

```diff
@@ -76,24 +60,19 @@ export class CircleGraphPanel extends CircleGraphCtrl {
 
     // Otherwise, create a new panel.
     const panel = vscode.window.createWebviewPanel(
-        CircleGraphPanel.viewType,
-        'CircleGraphPanel',
-        column || vscode.ViewColumn.One,
-        getWebviewOptions(extensionUri),
-    );
+        CircleGraphPanel.viewType, 'CircleGraphPanel', column || vscode.ViewColumn.One,
+        {retainContextWhenHidden: true});
```

Comment:
Shouldn't this be `CircleGraphCtrl.getWebviewOptions()` ?

Reply:
I think I wanted to do something with `getWebviewOptions` but I guess I'll do that in another PR.